tests/integration/awslambda/test_lambda_api.py

```diff
@@ -794,6 +794,75 @@ def test_lambda_image_and_image_config_crud(
             "get-function-config-response-after-delete-imageconfig", get_function_config_response
         )
 
+    @pytest.mark.aws_validated
+    def test_lambda_image_versions(
+        self, create_lambda_function_aws, lambda_su_role, test_image, snapshot, aws_client
+    ):
+        """Test lambda versions with package type image"""
+        image = test_image("alpine")
+        repo_uri = image.rpartition(":")[0]
+        snapshot.add_transformer(snapshot.transform.regex(repo_uri, "<repo_uri>"))
+        function_name = f"test-function-{short_uid()}"
+        create_image_response = create_lambda_function_aws(
+            FunctionName=function_name,
+            Role=lambda_su_role,
+            Code={"ImageUri": image},
+            PackageType="Image",
+            Environment={"Variables": {"CUSTOM_ENV": "test"}},
+            Publish=True,
+        )
+        snapshot.match("create-image-response", create_image_response)
```

Comment:
nit: any reason for alternating `-` and `_` 🤔

Reply:
Yes. I copied the first part of the test from one test, the second part of it from another, and then modified it, hence the different styles. Will fix!